docs/tutorials/multimodal/document/pdf_classification.ipynb

```diff
@@ -0,0 +1,234 @@
+    "from autogluon.multimodal import MultiModalPredictor\n",
+    "\n",
+    "predictor = MultiModalPredictor(label=\"label\", verbosity=5)\n",
+    "predictor.fit(\n",
+    "    train_data=train_data,\n",
+    "    hyperparameters={\"model.document_transformer.checkpoint_name\":\"microsoft/layoutlm-base-uncased\",\n",
+    "    \"optimization.top_k_average_method\":\"best\",\n",
+    "    },\n",
+    "    time_limit=120,\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate on Test Dataset\n",
+    "\n",
+    "You can evaluate the classifier on the test dataset to see how it performs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scores = predictor.evaluate(test_data, metrics=[\"accuracy\"])\n",
+    "print('The test acc: %.3f' % scores[\"accuracy\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Predict on a New PDF Document\n",
+    "\n",
+    "Given an example PDF document, we can easily use the final model to predict the label:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictions = predictor.predict({DOC_PATH_COL: [test_data.iloc[0][DOC_PATH_COL]]})\n",
+    "print(f\"Ground-truth label: {test_data.iloc[0]['label']}, Prediction: {predictions}\")\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If probabilities of all categories are needed, you can call predict_proba:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "proba = predictor.predict_proba({DOC_PATH_COL: [test_data.iloc[0][DOC_PATH_COL]]})\n",
+    "print(proba)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Extract Embeddings\n",
+    "\n",
+    "Extracting representation from the whole document learned by a model is also very useful. \n",
+    "We provide extract_embedding function to allow predictor to return the N-dimensional document feature where N depends on the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature = predictor.extract_embedding({DOC_PATH_COL: [test_data.iloc[0][DOC_PATH_COL]]})\n",
+    "print(feature[0].shape)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Other Examples\n",
+    "\n",
+    "You may go to [AutoMM Examples](https://github.com/autogluon/autogluon/tree/master/examples/automm) to explore other examples about AutoMM.\n",
+    "\n",
+    "## Customization\n",
+    "To learn how to customize AutoMM, please refer to :ref:`sec_automm_customization`.\n"
```

Comment:
You might need this to make the hyperlink work: `[Customize AutoMM](../advanced_topics/customization.ipynb)`